seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -219,10 +270,13 @@ angular.module('BE.seed.controller.mapping', [])
       $scope.setAllFieldsOptions = [{
         name: 'Property',
         value: 'PropertyState'
-      }, {
-        name: 'Tax Lot',
-        value: 'TaxLotState'
       }];
+      if (!$scope.mappingBuildingSync) {
+        $scope.setAllFieldsOptions.push({
+          name: 'Tax Lot',
+          value: 'TaxLotState'
+        })
+      }
```

Comment:
Rather than disabling the select, we manage the select options here.
This is necessary because a user might be mapping a bsync file, and NOT be working off of an existing preset, and needs to set the inventory type

Reply:
Should we do something similar for the admin page? As of now, the BE doesn't actually let the inventory type be changed, but the FE displays both as options.